doorstop/core/vcs/base.py

```diff
@@ -115,8 +115,11 @@ def paths(self):
                         log.debug(f"not skipping: {relpath}")
                         self._path_cache.append((path, filename, relpath))
                         continue
-                    # Skip hidden paths
-                    if os.path.sep + "." in os.path.sep + relpath:
+                    # Skip hidden paths except for 'remote folder'
```

Comment:
what if we want to include hidden files in references? For example, if we reference a CI file in a `.gitlab-ci` folder.

Reply:
you would still have the  `.unhidden` bit.

Reply:
https://github.com/CodethinkLabs/doorstop/pull/15/files#diff-5fa9aba911bf89d1fddf65b8e71f3a6b5340aeb9f14e056439a988060dc46983R114

Reply:
Doh! I missed that, because it wasn't shown in the 'changed' snippet.